experiments/tribler_idle_run/tribler_idle_run.py

```diff
@@ -1,49 +1,56 @@
 #!/usr/bin/env python3
 import os
+import signal
 from asyncio import ensure_future, get_event_loop, sleep
 from pathlib import Path
 
+from tribler_core.components.base import Session
 from tribler_core.config.tribler_config import TriblerConfig
-from tribler_core.session import Session
+from tribler_core.start_core import components_gen
 
 BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', '..'))
 
 
-class IdleTribleRunner:
+async def idle_tribler_run():
```

Comment:
**Question:** Besides the application tester, I don't think we have any Jenkins experiment that runs Tribler idle right? The would be a great validation test to see if we broken Gumby-Tribler support.

Reply:
We can use `tribler_idle_run` in a validation test, but I'm not sure we should. We also have `tribler_module` that can be used in a scenario file. In my understanding, we usually use Gumby to run scenarios, and in a scenario that uses `tribler_module`, we can use all that `tribler_idle_run` allows, and more. Maybe later we can remove `tribler_idle_run` entirely in favor of `tribler_module`. But you have much more experience with Gumby, and perhaps there are use-cases where `tribler_idle_run` looks preferable to `tribler_module`.

Reply:
It might indeed be better to remove the `tribler_run_idle` script since there currently is no use case for it. The only use-case I can think of is if we want to run a version of Tribler on our compute cluster (or other infrastructure) that is as close as possible to the end-user configuration. But we currently do not need that so I would recommend to remove it. Should we do so in this PR or a separate one?

Reply:
I think it is better to remove `tribler_idle_run` in a separate PR. This way, it will be easier to revert the removing commit and return `tribler_idler_run` if it later becomes useful again.